File: iPhone/VerBank-CSTSv3-IOSClientApi/CSTS3Comm/json/QuoteList.m
//
//  QuoteList.m
//  IOSClientApiTester
//
//  Created by felix on 7/25/13.
//  Copyright (c) 2013 allone. All rights reserved.
//

#import "QuoteList.h"
#import "JEDIDateTime.h"
#import "JEDISystem.h"

static NSString * gs_QList_jsonId  =@"QuoteList";
static NSString * gs_QList_sizes   =@"1";

@implementation QuoteList


//-------------------------------------------------------------
-(id)               init
{
    self = [super init];
    
    if(self != nil)
    {
        [super setEntry:@"jsonId" entry:gs_QList_jsonId];
    }
    
    return self;
}

//-------------------------------------------------------------
-(NSMutableArray *) getQuotes // QuoteData[]
{
    JEDI_SYS_Try
    {
        return [super getEntryArray:gs_QList_sizes];
    }
    JEDI_SYS_EndTry
    
    return nil;
}

//-------------------------------------------------------------
-(void)             setQuotes:(NSMutableArray *) quotes
{
    JEDI_SYS_Try
    {
        [super setEntry:gs_QList_sizes entry:quotes];
    }
    JEDI_SYS_EndTry
}

@end

File: WeChatService/EX02-CSTSv3-ClientAPI/src/jedi/ex01/CSTS3/comm/jsondata/QuoteList.java
package jedi.ex01.CSTS3.comm.jsondata;

import jedi.ex01.CSTS3.comm.struct.QuoteData;


public class QuoteList extends allone.json.AbstractJsonData {

	public static final String jsonId = "QuoteList";

	public static final String quotes = "1";

	public QuoteList(){
		super();
		setEntry("jsonId", jsonId);
	}

	public QuoteData[] getQuotes() {
		try {
			QuoteData[] data=getEntryObjectVec(QuoteList.quotes,new QuoteData[0]);
			return data;
		} catch (RuntimeException e) {
			return null;
		}
	}

	public void setQuotes(QuoteData[] quotes) {
		setEntry(QuoteList.quotes, quotes);
	}


}